test/unit/v1.0.0/converter-v1-to-v2.test.js

```diff
@@ -1279,6 +1279,306 @@ describe('v1.0.0 to v2.0.0', function () {
+                                            contentType: 'application/json'
+                                        },
+                                        formdata: {
+                                            contentType: 'multipart/form-data'
+                                        }
+                                    }
+                                },
+                                header: [],
+                                method: 'GET',
+                                url: 'https://postman-echo.com/get'
+                            },
+                            response: []
+                        }]
+                    });
+                    done();
+                });
+            });
+
+            it('should transform when dataOptions are not present', function (done) {
+                transformer.convert({
+                    id: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2',
+                    name: 'get-with-body',
+                    order: ['4f65e265-dd38-0a67-71a5-d9dd50fa37a1'],
+                    folders: [],
+                    folders_order: [],
+                    requests: [{
+                        id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                        headers: '',
+                        url: 'https://postman-echo.com/get',
+                        data: 'foo=bar',
+                        method: 'GET',
+                        dataMode: 'raw',
+                        collectionId: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2'
+                    }]
+                }, options, function (err, converted) {
+                    expect(err).to.not.be.ok;
+
+                    // remove `undefined` properties for testing
+                    converted = JSON.parse(JSON.stringify(converted));
+
+                    expect(converted).to.eql({
+                        info: {
+                            _postman_id: '84b2b626-d3a6-0f31-c7a0-47733c01d0c2',
+                            name: 'get-with-body',
+                            schema: 'https://schema.getpostman.com/json/collection/v2.0.0/collection.json'
+                        },
+                        item: [{
+                            _postman_id: '4f65e265-dd38-0a67-71a5-d9dd50fa37a1',
+                            name: '',
+                            request: {
+                                body: {
+                                    mode: 'raw',
+                                    raw: 'foo=bar'
+                                },
+                                header: [],
+                                method: 'GET',
+                                url: 'https://postman-echo.com/get'
+                            },
+                            response: []
+                        }]
+                    });
+                    done();
+                });
+            });
+
+            it('should transform body options {params to fromdata}', function (done) {
```

Comment:
This is no longer required as its already tested in the first test.

Instead, add a test for invalid `dataOptions` type.